opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -347,6 +347,12 @@ def wrapper(self, *args, **kwargs):
     return wrapper
 
 
+def _is_valid_link(context: SpanContext, attributes: types.Attributes) -> bool:
+    return bool(
```

Comment:
Nit: `bool` is unnecessary here.

Reply:
if `context.is_valid` is False you are returning `attributes` or `context.trace_state` that are not bool

Reply:
it's just to ensure return bool since the function might return truthy values that are not strictly boolean.

Reply:
Seems all tests are passing now. Thanks @ocelotl and @xrmx